### ISSUE TITLE
How can I get/view response headers?

### ISSUE THREAD

**Issue by IA21** [hendt/ebay-api#109]
I'm trying to use the Sell Feed API and working with the createTask/getTasks calls right now. I can't access the response headers, only the response json. Here's some relevant code:

```
eBay.sell.feed.createTask({
    feedType: 'LMS_ADD_FIXED_PRICE_ITEM',
    schemaVersion: '1149',
}).then((resp) => {
        console.log(resp);
}).catch((err) => {
        console.error(err);
});
```

The `resp` variable only contains response json, its not the actual axios reponse object which contains headers. `createTask` (and some other calls) do not return any response json, only a `Location` header but I can't figure out a way to access it. I am very new to node and typescript so it could very well be my lack of knowledge. I did somehow manage to print the headers to console using the response interceptor but I can't really make them part of the return object. I don't think this is how headers are actually supposed to be accessed. Its very much a cheap hack that did not work:

```
eBay.req.instance.interceptors.response.use((response) => {
    // works
    console.log({
        response: response,
        headers: response.headers,
    });
    // does not work
    return {
        response: response,
        headers: response.headers,
    };
});
```

**Comment by dantio** [comment]
I see... I'll work on this issue. Thank you for reporting it.

**Comment by IA21** [comment]
Could you suggest a temporary alternative method of accessing the headers with the sdk as it is right now? eBay LMS api will stop working at the end of April and I'm working on migrating those calls to the Sell Feed api. I'm thinking since I can access the headers in the interceptor, I can throw rest of my code there and perform the remaining work inside the interceptor. Let me know if there's a better alternative to that. The async/await paradigm is new to me so I might be missing obvious solutions.

**Comment by dantio** [comment]
I plan to fix the issue and Release today.

**Comment by dantio** [comment]
@IA21 
Just released the new version: `npm i ebay-api@7.0.0`.
How to use:
```js
eBay.sell.feed.api({returnResponse: true}).createTask({
    feedType: 'LMS_ADD_FIXED_PRICE_ITEM',
    schemaVersion: '1149',
}).then((resp) => {
  // access the response here
        console.log(resp);
}).catch((err) => {
        console.error(err);
});
```

Let me know if it works for you.

**Comment by IA21** [comment]
It works. I can now access headers alongside response body. Thanks! 🙂
